docs/accessing_resource_localhost.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Accessing your resource on localhost
```

Comment:
If you're using custom links, they're likely not on localhost. maybe "Accessing your resource" or "Surfacing links to your resource" or "Surfacing resource endpoints" or something

Reply:
Good call. Changed.